src/ralph/dns/dnsaas.py

```diff
@@ -98,7 +98,7 @@ def update_dns_records(self, record):
             return False
         return True
 
-    @lru_cache()
+    @cache()
```

Comment:
this cache here is useless since almost all record will be different. The only cache which makes sense here is caching based on `domain_name` param.